kotlin/workflow-testing/src/test/java/com/squareup/workflow/ChannelSubscriptionsIntegrationTest.kt

```diff
@@ -46,7 +46,7 @@ class ChannelSubscriptionsIntegrationTest {
    */
   private class SubscriberWorkflow(
     private val channel: Channel<String>
-  ) : StatefulWorkflow<Boolean, Boolean, ChannelUpdate<String>, (Boolean) -> Unit>() {
+  ) : StatelessWorkflow<Boolean, ChannelUpdate<String>, Unit>() {
```

Comment:
Makes this test workflow so much simpler!

Reply:
Nice